packages/cli/generators/project/templates/package.plain.json.ejs

```diff
@@ -71,7 +71,6 @@
     "@loopback/context": "<%= project.dependencies['@loopback/context'] -%>",
 <% if (project.projectType === 'application') { -%>
     "@loopback/core": "<%= project.dependencies['@loopback/core'] -%>",
-    "@loopback/dist-util": "<%= project.dependencies['@loopback/dist-util'] -%>",
```

Comment:
This change looks suspicious. For applications scaffolded with `loopbackBuild` feature enabled, the index file is using dist-util to load the correct `dist*` version (both in dev and in production), but `@loopback/dist-util` is not specified as a production dependency.

I think the following scenario will stop working:

```shell
# create an app and verify it works
$ lb4 app
$ npm it

# imagine we are deploying to production now
# and excluding node_modules directory
$ rm -rf node_modules

# now on the production box (or in Docker)
$ npm install --production
$ npm start
```

Could you please check?

IMO, we need to change the template to add dist-util when loopbackBuild is enabled. I believe this is needed for both application and extension projects.

Reply:
You probably didn't realize that we use `packages/cli/generators/project/templates/package.plain.json.ejs` when `loopbackBuild` is disabled.

Reply:
You are right, I did not realize we have two copies of `package.json` template. The proposed change makes sense now👍 

For the longer term, it would be great to unify these two templates to reduce the amount of duplicated information.